### PR TITLE
chore: Document flatten-library task

### PR DESCRIPTION
**Pull request by matz3**
Fixes: #302

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ The following tasks (known to this page) have been created and are available in
 | NPM package | Description | Badge |
 | ----------- | ----------- | ----- |
 | [ui5-task-babel](https://github.com/pwasem/ui5-task-babel#readme) | configurable transpiling using babel | [![npm version](https://badge.fury.io/js/ui5-task-babel.svg)](https://badge.fury.io/js/ui5-task-babel) |
+| [ui5-task-flatten-library](https://www.npmjs.com/package/ui5-task-flatten-library) | Prepares build result for deployment to SAP NetWeaver | [![npm version](https://badge.fury.io/js/ui5-task-flatten-library.svg)](https://badge.fury.io/js/ui5-task-flatten-library) |
```
Comment (vobu) [inline docs/index.md:26]: lower case `p` in "Prepares" (sorry 🤭)

File: packages/ui5-task-flatten-library/README.md [hunk 2]
```diff
@@ -59,4 +59,4 @@ builder:
 
 This work is [dual-licensed](../../LICENSE) under Apache 2.0 and the Derived Beer-ware License. The official license will be Apache 2.0 but finally, you can choose between one of them if you use this work.
 
-When you like this stuff, buy [@vobu](https://twitter.com/vobu) a beer or buy [@pmuessig](https://twitter.com/pmuessig) or [@TheVivekGowda](https://twitter.com/TheVivekGowda) a coke when you see them.
+When you like this stuff, buy [@vobu](https://twitter.com/vobu) or [@matthiaso](https://twitter.com/matthiaso) a beer or buy [@pmuessig](https://twitter.com/pmuessig) a coke when you see them.
```
Comment (vobu) [inline packages/ui5-task-flatten-library/README.md:62]: 👍

File: README.md [hunk 3]
```diff
@@ -20,6 +20,7 @@ packages
 ├── ui5-middleware-livetranspile    // middleware extension: on-demand es6 to es5 transpile when requesting js
 ├── ui5-middleware-simpleproxy      // middleware extension: simple express proxy
 ├── ui5-middleware-servestatic      // middleware extension: serve static resources
+├── ui5-task-flatten-library        // task extension: Prepares build result for deployment to SAP NetWeaver
```
Comment (vobu) [inline README.md:23]: please move 1 line down (alphabetical sorting of tasks) and lower-case "p" in "Prepares"

File: README.md [hunk 4]
```diff
@@ -82,6 +83,7 @@ yarn test:ci
 
 The tasks and middlewares developed in this project are also publicly available on NPM right here:
  
+* https://www.npmjs.com/package/ui5-task-flatten-library
```
Comment (vobu) [inline README.md:86]: please move 1 line down (alphabetical sorting of `npm` task modules)

File: README.md [hunk 5]
```diff
@@ -101,6 +103,7 @@ Available tasks in this project:
 
 | NPM package | Description | Badge |
 | ----------- | ----------- | ----- |
+| [ui5-task-flatten-library](https://www.npmjs.com/package/ui5-task-flatten-library) | Prepares build result for deployment to SAP NetWeaver | [![npm version](https://badge.fury.io/js/ui5-task-flatten-library.svg)](https://badge.fury.io/js/ui5-task-flatten-library) |
```
Comment (vobu) [inline README.md:106]: please move 1 line down (alphabetical sorting of task list)

**Review by vobu — CHANGES_REQUESTED** [review]
only minor spelling and layout/listing - sorry for the pedantry, but...ya know...

File: README.md (hunk 3 above)
Comment (matz3) [reply README.md]: I'd say `f` comes before `i`, right?

Comment (vobu) [reply README.md]: 😪 that's right...

**Review by vobu — APPROVED** [review]
LGTM

**Pull request merged**